tiledb/cc/CMakeLists.txt

```diff
@@ -0,0 +1,27 @@
+cmake_minimum_required(VERSION 3.20)
```

Comment:
This file is cruft and can be removed now that we have top-level setup.py support 😄